roles/zookeeper/tasks/upstream_install.yml

```diff
@@ -0,0 +1,34 @@
+- name: Create space for ZK Download
+  file:
+    state: directory
+    path: /usr/local/zk
+    mode: 0755
+
+- name: Download Upstream ZK Tarball
+  get_url:
+    dest: /usr/local/zk/zookeeper-3.4.10.tar.gz
+    url: https://apache.org/dist/zookeeper/stable/zookeeper-3.4.10.tar.gz 
+
+- name: Untar Zookeeper
+  unarchive:
+    src: /usr/local/zk/zookeeper-3.4.10.tar.gz
+    dest: /usr/local
+    remote_src: yes
```

Comment:
is this idempotent? personal opinion, i'd try and find an rpm.

Reply:
It is because it is version specific. RPMs would require repos, and I don't have one that is reliable. You got one? I'll fix this happily!